### PR TITLE
15 - reproducible repo

### PR DESCRIPTION
**Pull request by mxochicale**
# Addressing #15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,42 @@
...
+## Maintainers
+* [Dave Cash](https://github.com/davecash75)
+* [Jamie Mcclelland](https://github.com/jamie-mcclelland)
+
+## Contributors
+This project exists thanks to all the people who contribute.
+* [Milan Malfait](https://github.com/milanmlft)
+* [Zakaria Senousy](https://github.com/zsenousy)
+* [Stephen Thompson](https://github.com/thompson318)
+* [Miguel Xochicale](https://github.com/mxochicale)
```
Comment (thompson318) [inline README.md:34]: Can you add Dave and Jamie?

**Review by thompson318 — COMMENTED** [review]
A couple of small comments.

File: README.md (hunk 1 above)
Comment (mxochicale) [reply README.md:34]: Already there as maintainers https://github.com/HealthBioscienceIDEAS/Medical-Image-Registration-Short-Course/tree/15-reproducible-repo?tab=readme-ov-file#maintainers

**Comment by mxochicale** [comment]
Thanks @zsenousy and @thompson318 for your comments, I think have resolved them! Let me know if you have more otherwise please approve this PR so we can merge it to main! Thanks, Miguel

**Review by thompson318 — APPROVED** [review]
Looks good, thank you.

**Pull request merged**